### PR TITLE
Visual studio server implemented

### PR DESCRIPTION
**Pull request by drqc**
This would add vscode server to the ace box. I would like to sync tomorrow to explore options for seeding variables to replace default options in roles to allow changesc in the vscode folder from the tfvars

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,4 @@ override.tf.json
 
 # Nodejs
 *.env*
+terraform/gcloud/home/diego/.ssh/id_rsa
```
Comment (kristofre) [inline .gitignore]: @drqc no references to your personal folders. Please change this to a generic folder

Comment (drqc) [reply .gitignore]: sure, fixed

File: user-skel/ansible/roles/config/defaults/main.yml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 ---
-features_enabled_default: ["feature_oneagent", "feature_activegate", "feature_gitea", "feature_jenkins", "feature_dashboard", "feature_keptn"]
+features_enabled_default: ["feature_oneagent", "feature_activegate", "feature_gitea", "feature_jenkins", "feature_dashboard", "feature_keptn", "feature_vscodeserver"]
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/config/defaults/main.yml]: Please add "features_enabled_\<your use case\>"

Comment (kristofre) [reply user-skel/ansible/roles/config/defaults/main.yml]: Would this need to be enabled by default? Want to keep an eye on resource consumption etc. Unless we are doing a HOT session users will most likely not need this to do the standard use cases.

**Comment by tobigremmer-dt** [comment]
Hey @drqc , if I remember correctly, these implementations were made for a Perform HOT session. Eventually the session didn't use VSC. Is this still needed?

**Comment by tobigremmer-dt** [comment]
Closing this PR for now. In case this feature is needed in the future, please resolve any conflicts and create a new PR.

**Comment by drqc** [comment]
Thanks Tobi I missed this message. You are correct this was for a HOT session. I would reuse this but in a custom use case

**Pull request closed without merging**